nexus-repository-cpan-it/src/test/java/org/sonatype/nexus/repository/cpan/internal/CpanProxyIT.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.cpan.internal;
+
+import org.sonatype.goodies.httpfixture.server.fluent.Behaviours;
+import org.sonatype.goodies.httpfixture.server.fluent.Server;
+import org.sonatype.nexus.pax.exam.NexusPaxExamSupport;
+import org.sonatype.nexus.repository.Repository;
+import org.sonatype.nexus.repository.http.HttpStatus;
+import org.sonatype.nexus.testsuite.testsupport.FormatClientSupport;
+import org.sonatype.nexus.testsuite.testsupport.NexusITSupport;
+
+import org.hamcrest.MatcherAssert;
+import org.junit.Test;
+import org.ops4j.pax.exam.Configuration;
+import org.ops4j.pax.exam.Option;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.sonatype.goodies.httpfixture.server.fluent.Behaviours.content;
+import static org.sonatype.goodies.httpfixture.server.fluent.Behaviours.error;
+import static org.sonatype.nexus.testsuite.testsupport.FormatClientSupport.status;
+
+public class CpanProxyIT
+    extends CpanITSupport
+{
+  private static final String TEST_PATH = "authors/id/E/EH/EHUELS/Test-Dependencies-0.24.tar.gz";
```

Comment:
Name of the variable has general meaning. Usually, we give more specific names relates to its value.

Reply:
In: 5f4f849